homeassistant/components/cover/template.py

```diff
@@ -369,9 +386,12 @@ def async_update(self):
             except ValueError as ex:
                 _LOGGER.error(ex)
                 self._tilt_value = None
-        if self._icon_template is not None:
+
+        icon_template = (self._icon_template if self._icon_template else
```

Comment:
Please just write normal if statements, by getting them multiline it is very difficult to read.